Multisig/UI/Safe Management/EnterSafeNameView.swift

```diff
@@ -9,13 +9,72 @@
 import SwiftUI
 
 struct EnterSafeNameView: View {
+    var address: String? = "0xAB3e244863e1a127333aBa15235aD50E0954146F"
+    @State private var name: String = ""
     var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
+        NavigationView {
+            VStack(spacing: 0) {
+                Identicon(address!).frame(width: 40, height: 40, alignment: .center)
+                HStack (alignment: .top) {
+                    Image("ico-circle-check")
+                    AddressText(address!).multilineTextAlignment(.center)
+                }
+                .padding(.trailing, 37)
+                .padding(.top, 9)
+                Text("Choose a name for the Safe. The name is only stored locally and will not be shared with Gnosis or any third parties.")
```

Comment:
strange space after this line, I would instead have a blank line between each element

```
Identicon(...)

HStack { 
...
}

Text

VerfiableTextField

Spacer
```